vmdb/lib/miq_automation_engine/models/miq_ae_class_copy.rb

```diff
@@ -78,15 +78,14 @@ def add_fields
 
   def validate
     dest_class = MiqAeClass.find_by_fqname("#{@target_ns_fqname}/#{@target_name}")
-    if dest_class
-      dest_class.destroy if @overwrite
-      raise "Destination Class already exists #{dest_class.fqname}" unless @overwrite
-    end
+    return unless dest_class
+    dest_class.destroy if @overwrite
```

Comment:
@Fryguy @kbrock 
The rubocop error is
Use a guard clause instead of wrapping the code inside a conditional expression.  81,1 

79   def validate
   80     dest_class = MiqAeClass.find_by_fqname("#{@target_ns_fqname}/#{@target_name}")
S> 81     if dest_class
   82       dest_class.destroy if @overwrite
   83       raise "Destination Class already exists #{dest_class.fqname}" unless @overwrite
   84     end
   85   end